modules/ROOT/pages/application-deployment.adoc

```diff
@@ -4,11 +4,15 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: deploy, application, applications, runtime, amc, cloudhub, on premises, on premise
 
+Learn how to deploy, undeploy and update Mule runtime engine (Mule) applications.
```

Comment:
Mule runtime engine (Mule) applications > Mule applications. I think this is the only form we use for apps. Please confirm in glossary.